v3/src/components/graph/adornments/count/count-adornment-component.tsx

```diff
@@ -60,27 +72,36 @@ export const CountAdornment = observer(function CountAdornment(props: IAdornment
   }, [defaultFontSize, fontSize, graphModel.adornmentsStore, plotWidth, textContent])
 
   const plotCaseCounts = useCallback(() => {
+    // If the graph's points have been grouped into bins, we need to show the case count within each bin.
+    //
     // If there are movable values present, we need to show the case count within each sub-plot region defined by the
     // movable values and the min and max of the primary axis. For example, if there is one movable value, the sub-plot
     // will have two regions, one from the axis' min value to the movable value, and another from the movable value
     // to the axis' max value.
+    //
+    // It should not be possible to have both bin boundaries and movable values present at the same time.
 
     if (subPlotRegionBoundariesRef.current.length < 3 || graphModel.plotType !== "dotPlot") {
-      // If there are no movable values present, we just show a single case count.
+      // If there are no bin boundaries or movable values present, we just show a single case count.
       setDisplayCount(<div>{textContent}</div>)
       return
     }
 
     const regionCountParams: IRegionCountParams = {
       cellKey,
       dataConfig,
+      inclusiveMax: graphModel.pointDisplayType !== "bins",
```

Comment:
Worth a comment here. I take it the idea is that movable value counts are inclusive while bin counts are exclusive? I trust that's the v2 behavior? A priori I might have expected both to be exclusive.

Reply:
@kswenson I think it's true only regions defined by movable values are inclusive. Maybe I'm thinking about it wrong and/or using the wrong terminology, though. Below is a screenshot from V2. With movable values, if a point has the same value as one of the movable values, it's included in the count for the region whose upper boundary is defined by that movable value. With bins, if a point's value matches a bin boundary, it's placed in the bin whose lower boundary is defined by that bin. Does that all seem to make sense with the use of `inclusiveMax` here? Or do you think it needs to be adjusted somehow?

<img width="1741" alt="image" src="https://github.com/concord-consortium/codap/assets/367459/86378f35-d3c0-45fd-aea4-20a92c845e31">
(My mouse cursor is hovering over the point in the "50,60" bin in the rightmost graph to show that its value is 50.)

Reply:
I was just suggesting a comment, which you've added, and making sure you'd checked the v2 behavior, which it seems you have.